flink-doris-connector/src/main/java/org/apache/doris/flink/serialization/RowBatch.java

```diff
@@ -330,26 +378,9 @@ public boolean doConvert(
                 }
                 break;
             case "DATETIME":
-                if (!minorType.equals(Types.MinorType.TIMESTAMPMICRO)
-                        && !minorType.equals(Types.MinorType.VARCHAR)) {
-                    return false;
-                }
-                if (minorType.equals(Types.MinorType.VARCHAR)) {
-                    VarCharVector varCharVector = (VarCharVector) fieldVector;
-                    if (varCharVector.isNull(rowIndex)) {
-                        addValueToRow(rowIndex, null);
-                        break;
-                    }
-                    String stringValue = new String(varCharVector.get(rowIndex));
-                    LocalDateTime parse = LocalDateTime.parse(stringValue, dateTimeFormatter);
-                    addValueToRow(rowIndex, parse);
-                } else {
-                    LocalDateTime dateTime = getDateTime(rowIndex, fieldVector);
-                    addValueToRow(rowIndex, dateTime);
-                }
-                break;
```

Comment:
Will there be any problem if delete it here？because the formatter of datetime is different from that of datetimev2